generators/blazor/templates/src/client/Project.Client/Pages/Entities/_entityClass_/_entityClass_Detail.razor.ejs

```diff
@@ -5,16 +5,16 @@ const entityClassName = pascalizedEntityClass;
 const entityVariableName = camelCasedEntityClass;
 const entityClassNamePluralized = pascalizedEntityClassPlural;
 _%>
-@page "/<%= lowerCasedEntityClass %>/{Id<%_ if(primaryKeyType !== 'string') { %>:<%= primaryKeyType %> <%_ } _%> }/view"
+@page "/<%= lowerCasedEntityClass %>/{Id<%_ if(isNumericPrimaryKey(primaryKeyType)) { %>:<%= primaryKeyType %> <%_ } _%> }/view"
```

Comment:
here we need the id type like "{Id:guid}" without this the detail page can't convert the id